backend/endpoints/messages.py

```diff
@@ -187,7 +188,7 @@ async def update_message(
         )
 
     message["richUiData"] = payload["richUiData"]
-    payload["edited"] = True
+    payload["edited"] = message["edited"] = True
     edited_message = await DB.update(
         MESSAGE_COLLECTION, document_id=message_id, data=payload
```

Comment:
```suggestion
        MESSAGE_COLLECTION, document_id=message_id, data=message
```
Because `message` is now updated